.travis.yml

```diff
@@ -32,6 +32,7 @@ install:
 script:
   - if [[ $TESTS == true ]]; then source continuous_integration/travis/run_tests.sh ; fi
   - if [[ $LINT == true ]]; then pip install flake8 ; flake8 distributed ; fi
+  - if [[ $LINT == true ]]; then pip install black; black distributed ; fi
```

Comment:
This will format the files and only returns a non-zero exit code if there's an internal error. You probably want `--check` instead to check that they pass.